.install/ubuntu_18.04.sh

```diff
@@ -6,10 +6,21 @@ MODE=$1 # whether to install using sudo or not
 $MODE apt update -qq
 $MODE apt upgrade -yqq
 $MODE apt dist-upgrade -yqq
-$MODE apt install -yqq software-properties-common
+$MODE apt install -yqq software-properties-common unzip rsync
+
 $MODE add-apt-repository ppa:ubuntu-toolchain-r/test -y
 $MODE add-apt-repository ppa:git-core/ppa -y
 $MODE apt update
-$MODE apt install -yqq git wget make gcc-10 g++-10 openssl libssl-dev
+$MODE apt-get install -yqq build-essential git wget make gcc-10 g++-10 openssl libssl-dev
 $MODE update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 60 --slave /usr/bin/g++ g++ /usr/bin/g++-10
+
+# Install Python 3.7
+$MODE apt-get install python3.7 python3.7-venv python3-venv python3.7-dev python3-dev -y
```

Comment:
why 3.7? we already passed its end of life

Reply:
didn't know that